app/models/custom_attribute.rb

```diff
@@ -2,7 +2,17 @@ class CustomAttribute < ApplicationRecord
   belongs_to :resource, :polymorphic => true
   serialize :serialized_value
 
+  def value=(value)
```

Comment:
I think its worthwhile to save that string value so if we would want to query based on it value it will be easier, in case querying isn't needed we might be able to remove it along the way.